lighthouse-core/lib/stack-packs.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2010 Google Inc. All Rights Reserved.
```

Comment:
> Copyright 2010

whoa where'd you copy *this* file from 😆

Reply:
haha I actually changed it and thougth I typed a 9 :p oh well mistake were made 😄